lib/scholar/covariance.ex

```diff
@@ -29,39 +29,61 @@ defmodule Scholar.Covariance do
   @doc """
   Computes covariance matrix for sample inputs `x`.
 
+  #{~S'''
+  The value on the position $(i, j)$ in matrix is caluculated using formula:
+  $$ Cov(X\_{i}, X\_{j}) = \frac{\sum\_{k}\left(x\_{k} - \bar{x}\right)\left(y\_{k} - \bar{y}\right)}{N - 1} $$
+  Where:
+
+    * $X\_{i}$ is a $i$th row of input
+
+    * $x\_{k}$ is a $k$th value of $X_{i}$
+
+    * $y\_{k}$ is a $k$th value of $X_{j}$
+
+    * $\bar{x}$ is the mean of $X\_{i}$
+
+    * $\bar{y}$ is the mean of $X\_{j}$
+
+    * $N$ is the number of samples
+
+  This is a non-biased version of covariance. The biased version has $N$ in denominator instead of $N - 1$.
+  '''}
```

Comment:
I don't know how to format this block correctly. Ctrl + Shift + I doesn't work for this one.

Reply:
This seems fine to me :)

Reply:
Yeah, but the mix formatter fails on this one I guess.

Reply:
When I run `mix format --check-formatted` on my local repo everything is fine, but here it fails on CI.  😞

Reply:
Are you using the same Elixir version that CI is using?

Reply:
Fixed it. 1.13 formatter sees something that 1.14 doesn't